cluster/addons/fluentd-gcp/fluentd-gcp-configmap.yaml

```diff
@@ -83,7 +81,19 @@ data:
     <match reform.**>
       @type record_reformer
```

Comment:
NOTE: This whole `record_reformer` section sets up the `local_resource_id` for `k8s_container` resource. It's present in `fluentd-gcp-configmap.yaml` only (not `fluentd-gcp-configmap-old.yaml`).